### PR TITLE
Oriented views

### PR DESCRIPTION
**Pull request by floryst**
This PR rewrites the views to better handle dataset orientation.
- cameras transform to the dataset, not the other way around
- slice views now mimic an orthogonal MPR view, so there should be consistent coronal, sagittal, and axial views (assuming the  dataset orientation is correct)
  - This means the slices are along IJK, but each slice is correctly oriented in space.

### CODE REVIEW

File: src/components/VtkThreeView.vue [hunk 1]
```diff
@@ -42,9 +42,9 @@ export default {
 
     const {
       sceneSources,
-      worldOrientation,
+      imageConfig,
```
Comment (timthirion) [inline src/components/VtkThreeView.vue:45]: How about `imageParams`? `imageConfig` makes me thing of a software-level configuration for images, not image metadata.

I realize this is a big search and replace but may make things clearer overall.

**Review by timthirion — COMMENTED** [review]
Code LGTM. One minor comment on naming.

File: src/components/VtkThreeView.vue (hunk 1 above)
Comment (floryst) [reply src/components/VtkThreeView.vue:45]: +1 I agree that `imageParams` is clearer than `imageConfig`.

**Comment by timthirion** [comment]
Code LGTM but I haven't tested. Let me know if you'd like me to before merging.

**Review by timthirion — APPROVED** [review]

**Pull request merged**